pyop2/global_kernel.py

```diff
@@ -439,7 +439,7 @@ def vectorise(self, wrapper, iname, batch_size):
         # }}}
 
         # split iname
-        slabs = (1, 1)
+        slabs = (0, 1)
```

Comment:
The loop always starts from 0 now because we have shifted the original loop index to `shifted_iname` right?

Reply:
Yep!